tests/unit/completion/test_completer.py

```diff
@@ -0,0 +1,136 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2016 Ryan Roden-Corrent (rcorre) <ryan@rcorre.net>
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests for the Completer Object."""
+
+from unittest.mock import Mock
```

Comment:
This might be a pet-peeve of mine, but I really prefer always importing modules, and not objects from modules - in this case, simply `import unittest.mock` and use `unittest.mock.Mock`.

(I don't do that for Qt types, as they're at least easily distinguishable from things in the current module due to the leading `Q`)